CHANGELOG-v3.adoc

```diff
@@ -12,58 +12,45 @@ For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim
 *_General_*
 
 - Merge 3.16.1 https://github.com/gravitee-io/issues/issues/7268[#7268]
+- Merge 3.16.2 https://github.com/gravitee-io/issues/issues/7402[#7402]
 - Request does not return the good path https://github.com/gravitee-io/issues/issues/7220[#7220]
```

Comment:
```suggestion
- [debug mode] Request does not return the good path https://github.com/gravitee-io/issues/issues/7220[#7220]
```

Reply:
maybe we can move it in this section ?
https://github.com/gravitee-io/issues/pull/7403/files#diff-4d2bf6d0e64bb639b2173df19100ec5a04d076270b4e32c9d14a98c046d09126R20

Reply:
actually, the issue is not well defined. Is it a UAT or a bug ?
if it is a UAT, we have to remove it.
if it is a bug, it should be in a "Console" section, but in the "bug fix" part